### DISCUSSION TITLE
Proper Syntax for "noindex" property on docs.json configuration

### DISCUSSION THREAD

**Discussion by jwafu** [invertase/docs.page#301] [Q&A]
In the documentation there is a small section on utilizing the "noindex" property:

![Screenshot 2023-04-10 at 10 59 34 AM](https://user-images.githubusercontent.com/55861512/230927528-ddf68661-a34c-419f-a7e2-e6610c1aa503.png)

Considering (from my findings so far) most sites that use docs.page want to be crawled and indexed, Im having a hard time finding an example. Also am curious if there is a reason why this property is a string and not a boolean value - could make it easier to infer the meaning with this property. 👍

**Comment by alexdukeinvertase** [comment]
Hey, sorry for the delay - this was resolved in v2. `noindex` is now a boolean under `seo`, not a top-level string:

```json
{
  "seo": {
    "noindex": true
  }
}
```

Default is `false`. That sets `noindex` on every page and updates `robots.txt`. For a single page, use frontmatter `noindex: true` instead. Old v1 configs with a top-level noindex field still work, but they map across to seo.noindex now. 

More here if you need: https://docs.page/invertase/docs.page/features/search-engine-indexing
